rotkehlchen/db/schema.py

```diff
@@ -823,7 +823,8 @@
     endpoint TEXT NOT NULL,
     owned INTEGER NOT NULL CHECK (owned IN (0, 1)),
     active INTEGER NOT NULL CHECK (active IN (0, 1)),
-    weight INTEGER NOT NULL
+    weight INTEGER NOT NULL,
+    blockchain TEXT NOT NULL DEFAULT "ETH"
```

Comment:
do we need the default? Since we will be always inputting it from now on? Being explicit is much better than implicit

Reply:
I also wondered about it while writing it. I've removed the default 'eth'.